recipes/libcurl/all/conanfile.py

```diff
@@ -260,17 +278,19 @@ def _build_with_autotools(self):
         with tools.environment_append(env_run.vars):
             with tools.chdir(self._source_subfolder):
                 use_win_bash = self._is_mingw and not tools.cross_building(self.settings)
-                autotools, autotools_vars = self._configure_autotools()
 
                 # autoreconf
                 self.run('./buildconf', win_bash=use_win_bash)
 
                 # fix generated autotools files
-                tools.replace_in_file("configure", "-install_name \\$rpath/", "-install_name ")
+                # tools.replace_in_file("configure", "-install_name \\$rpath/", "-install_name ")
```

Comment:
This is needed only for macOS. We can guard it with `if` but it is not nessesary because it does not affect other platforms. The command makes the binary relocatable without hardcoding absolute path from the build machine. You can check the binary with `otool` with or without this patch.

Reply:
ok, thanks for the info! 
will do that tomorrow morning